xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3647,6 +3645,14 @@ void CVideoPlayer::SetSpeed(float speed)
   
   m_newPlaySpeed = speed * DVD_PLAYSPEED_NORMAL;
   SetPlaySpeed(speed * DVD_PLAYSPEED_NORMAL);
+  if (m_newPlaySpeed == DVD_PLAYSPEED_NORMAL && m_playSpeed != DVD_PLAYSPEED_NORMAL)
```

Comment:
this block needs to be before SetSplaySpeed

with an outer condition and simpler inner conditions the code would be more readable
what do you think?

```
if (m_newPlaySpeed != m_playSpeed)
{
  if (m_newPlaySpeed == DVD_PLAYSPEED_NORMAL)
    m_callback.OnPlayBackResumed();
  else if (m_newPlaySpeed == DVD_PLAYSPEED_PAUSED)
    m_callback.OnPlayBackPaused();
}
SetPlaySpeed(speed * DVD_PLAYSPEED_NORMAL);
```